### ISSUE TITLE
an error in run-sdm.R: Cannot create a RasterLayer object from this file.

### ISSUE THREAD

**Issue by kensaku-okada** [jcoliver/ebutterfly-sdm#3]
Hello Jeff.

I tried to run the new "run-sdm.R" in my local env (after running get-observation-data.R of cour), but got the following error.
----------------------
Error in .local(.Object, ...) : 
Error in .rasterObjectFromFile(x, band = band, objecttype = "RasterLayer",  :
 Cannot create a RasterLayer object from this file.
Calls: SDMBioclim ... raster -> raster -> .local -> .rasterObjectFromFile
Execution halted
----------------------

In debugging, I detected this error occurs in the following line in run-sdm.R , but it looks too difficult for me to detect and fix the bug.

species.rasters <- SDMBioclim(data = obs.data, bg.replicates = bg.replicates)

Could you tell me how to fix it, or you fix this bug???

**Comment by jcoliver** [comment]
I could not replicate this error. Would you mind sending me (via e-mail):

1. The command you issued from the terminal to run the script; it should be something like: `Rscript --vanilla run-sdm.R data/inaturalist/60606-iNaturalist.csv 60606 output/ 10 0.5`
2. The input file you used

**Comment by jcoliver** [comment]
From K. Okada:

> This is the command I ran. It is same as shown in README.md
> `Rscript --vanilla run-sdm.R data/inaturalist/60606-iNaturalist.csv 60606 output/ 50 0.5`

The script, from the command above, is being run from _within_ the `scripts` directory, but it should instead be run from the top-level folder like:
`Rscript --vanilla scripts/run-sdm.R data/inaturalist/60606-iNaturalist.csv 60606 output/ 50 0.5`

I have updated the README.md with updated path information.

**Comment by kensaku-okada** [comment]
![image](https://user-images.githubusercontent.com/6435299/33348997-8cf44576-d455-11e7-9d91-29706daaddc0.png)
Thank you for your response. I succeeded to get the sdm result. The cause of this problem would be I ran run-sdm.R in scripts directory (maybe I changed the original command to run it in scripts directory).
Now I understand I need to run both get-observation-data.R and run-sdm.R in ebutterfly-sdm directory.
